docs/examples/form_grid/README.md

```diff
@@ -1,13 +1,11 @@
-# Form
+# FormGrid
```

Comment:
Renamed the `Form` example to `FormGrid`, as this example is mainly about the layout. And down the line I might add another `Form` component whose logic would be mainly for being able to pass initial form data as a simple dict.